Resources/Maps/_Sunrise/Nonstations/nukieplanet.yml

```
@@ -11531,7 +11524,7 @@ entities:
       inSlot: head
     - type: Physics
       canCollide: False
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)